controllers/application_controller.php

```diff
@@ -5,8 +5,8 @@
 // ======= CONSTANTS CONFIGURABLES =======
 define('HOST_DB', 'localhost');
 define('NAME_DB', 'foroweb');
-define('USER_DB', 'root');
-define('PASS_DB', ' ');
+define('USER_DB', 'mariana');
+define('PASS_DB', 'contraseña');
```

Comment:
```suggestion
define('USER_DB', 'root');
define('PASS_DB', '');
```
do not leave your personal environment configs on the repository